Riot/Modules/TabBar/TabBarCoordinator.swift

```diff
@@ -1034,13 +925,13 @@ final class TabBarCoordinator: NSObject, TabBarCoordinatorType {
             return
         }
 
-        masterTabBarController.homeViewController.startActivityIndicator()
+//        masterTabBarController.homeViewController.startActivityIndicator()
```

Comment:
Why did you remove the indicator ?

Reply:
There is no homeViewController in Tchap. So I forgot to add the tchap comment. I will add it.